source/io/library_propertyNode.js

```diff
@@ -1,13 +0,0 @@
-(function () {
-    'use strict';
-    var propertyNodeCAPI = {
-        jsPropertyNodeWrite: function () {
-            Module.propertyNode.jsPropertyNodeWrite.apply(undefined, arguments);
```

Comment:
Need to delete the corresponding https://github.com/ni/VireoSDK/blob/master/source/io/module_propertyNode.js

Reply:
And the reference to it in https://github.com/ni/VireoSDK/blob/master/source/core/vireo.loader.shared.js#L6

Reply:
Done. Thanks for pointing out this file